bioblend/galaxy/histories/__init__.py

```diff
@@ -76,23 +76,28 @@ def show_history(self, history_id, contents=False, deleted=None, visible=None, d
         :param history_id: Encoded history ID to filter on
 
         :type contents: bool
-        :param contents: When ``True``, the complete list of datasets in the
-          given history.
+        :param contents: When ``True``, return also the complete list of
```

Comment:
It's not `also`, it's a different output altogether.
```suggestion
        :param contents: When ``True``, return the complete list of
```

Reply:
Uh, that's right! I've made it even more explicit in a rebase of the last commit.